frontend/src/dsnpLink.ts

```diff
@@ -200,7 +201,7 @@ export type AuthMethods = {
 export function configureAuth(params?: r.CreateContextParams<AuthMethods>["authProviders"]): AuthMethods {
     return { tokenAuth: params?.tokenAuth && new r.HttpBearerSecurityAuthentication(params.tokenAuth) };
 }
-
+// TODO: Was this file edited manually to add REACT_APP_BACKEND_URL?
```

Comment:
yes... is this a generated file?

Reply:
Yes, `dsnpLink.ts` is generated from `openapi.json`, see frontend `npm run gen:client`

Reply:
See, this is the problem with generated code. We'll have to revisit the whole question of tooling around this app template. The backend is no longer generated from `openapi.json`, so we may encounter drift. This is probably a post-1.0 thing, but this setup is very brittle & should be addressed.